precompile/contracts/rewardmanager/contract.go

```diff
@@ -45,34 +45,31 @@ var (
 	//go:embed contract.abi
 	RewardManagerRawABI string
 
-	RewardManagerABI        abi.ABI                                // will be initialized by init function
-	RewardManagerPrecompile precompile.StatefulPrecompiledContract // will be initialized by init function
+	RewardManagerABI        = parseABI(RewardManagerRawABI)
+	RewardManagerPrecompile = createRewardManagerPrecompile() // will be initialized by init function
```

Comment:
can we move these calls into `init()` so that it's more clear they will be called in order?

Reply:
We're already using `init` in module files. Using `init` here would register the module with a nil contract.